technical/indicators.py

```diff
@@ -785,9 +785,18 @@ def vpci(dataframe, period_short=5, period_long=20):
     return vpci
 
 
-def williams_percent(dataframe):
+def williams_percent_deprecated(dataframe):
```

Comment:
Considering the danger of this implementation (not rolling - using future data during backtesing) - i would remove this function completely and replace it with the new implementation.

If someone REALLY want's to use this, then using it directly from `pyti` is an option.

Reply:
ok